modules/node/src/appRegistry/appRegistry.repository.ts

```diff
@@ -5,10 +5,22 @@ import { AppRegistry } from "./appRegistry.entity";
 @EntityRepository(AppRegistry)
 export class AppRegistryRepository extends Repository<AppRegistry> {
   async findByNameAndNetwork(name: string, chainId: number): Promise<AppRegistry> {
-    return this.findOne({ where: { name, chainId } });
+    return this.findOne({
```

Comment:
Interesting, these AppRegistry entities are essentially config objects. Maybe it would be better if they were loaded into memory on startup? We don't expect these to ever change at runtime.